app/graphql/mutations/send_push_notification.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Mutations
+  class SendPushNotification < BaseMutation
+    argument :title, String, required: true
+    argument :body, String, required: false
+    argument :force_create, Boolean, required: false
+
```

Comment:
kann man sonst nicht noch ein Object/url mitgeben die dann als direktLink in die App hinein führt (zu einer News oder was auch immer) wenn man auf die Push klickt?

Reply:
Nein, Direktlink funktioniert anders, dafür müssen `data: query_type` und `data: id` mitgegeben werden. Das geht schon per Main-Server, brauchen wir hier aber nicht.